auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -715,7 +716,7 @@ def get_train_loader(self, data, cache_rate=0, persistent_workers=False):
         if cache_rate > 0:
             runtime_cache = self.get_shared_memory_list(length=len(data))
             train_ds = CacheDataset(
-                data=data, transform=train_transform, copy_cache=False, cache_rate=cache_rate, runtime_cache=runtime_cache
+                data=data, transform=train_transform, copy_cache=False, cache_rate=cache_rate, runtime_cache=runtime_cache, num_workers=self.config["num_cache_workers"]
             )
         else:
             train_ds = Dataset(data=data, transform=train_transform)
```

Comment:
because we're using runtime_cache in segresnet,   the "num_cache_workers" option will never be used.   It's only used when pre-caching the whole dataset.   So exposing this option to a user (for segresent) in hyper_parameters.yaml - is misleading, since it won't do anything.    And since it's doesn't do anything, I don't think we need to add this option all to segresnet.

Reply:
@myron @dongyang0122 how about we keep `num_cache_workers` for the other 3 algorithms and segresnet continues to use `runtime_cache`? Or we better just drop this @wyli

Reply:
I'm ok, if you do this only for other 3  algs

Reply:
I removed `num_cache_workers` from segresnet for now.